Sming/Libraries/.patches/IR.patch

```diff
@@ -0,0 +1,511 @@
+diff --git a/src/IRrecv.cpp b/src/IRrecv.cpp
+index 38e2c77..aa1cdbb 100644
+--- a/src/IRrecv.cpp
++++ b/src/IRrecv.cpp
+@@ -5,80 +5,22 @@
+ 
+ #include "IRrecv.h"
+ #include <stddef.h>
+-#ifndef UNIT_TEST
+ extern "C" {
+-  #include <gpio.h>
+-  #include <user_interface.h>
++#include <gpio.h>
++#include <user_interface.h>
+ }
+ #include <Arduino.h>
+-#endif
++
+ #include <algorithm>
+ #include "IRremoteESP8266.h"
++#include "Print.h"
+ 
+-#ifdef UNIT_TEST
```

Comment:
Is there a reason for the big amount of changes in that file? I would prefer if those changes are improvements to be submitted directly to the upstream project and to keep here only the small changes that make the code compile.